README.md

```diff
@@ -19,6 +19,8 @@ Make sure you have installed the node version and yarn version specified in
 
 **Tip:**  You might want to use [nvm:  Node Version Manager](https://github.com/nvm-sh/nvm) and [yvm:  Yarn Version Manager](https://yvm.js.org/docs/overview) to switch between versions more easily.
 
+**Tip:** There are some known hiccups running the frontend on Windows see [run on Windows section](#Run-on-Windows)
```

Comment:
The hiccups are are actually only related to running the tests, and also only related to Ubuntu running in Windows via WSL.  Suggest that we move the this text to the test section, and modify to say:  There are some known hiccups running the frontend tests on Ubuntu via Windows Subsystem for Linux (WSL).  For details, see [run on WSL section](#Run-on-WSL)